.github/workflows/build_and_deploy_ext2_img.yml

```diff
@@ -0,0 +1,119 @@
+name: Create and deploy ext2 image to pages
+
+# Controls when the workflow will run
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+  # Lets you supply arguments when you run a github action manually.
+    inputs:
+      dockerfile_context_path:
+        description: 'Path or url to the Dockerfile its context.'
+        required: true
+        default: 'dockerfiles'
+
+      dockerfile_path:
+        description: 'Path to the Dockerfile.'
+        required: true
+        default: 'dockerfiles/Dockerfile'
+  
+      image_size:
+        description: 'Size of the base ext2 image. 1G is the max and recommended size. Reference the dd manpage for multiplicative suffixes'
+        required: true
+        default: '1G'
```

Comment:
GH pages says that the max for the _whole_ setup is 1GB, so we cannot make the image itself 1GB as we won't have space for anything else. I am unsure if GH actually enforces the 1GB limit or it is in practice a little bit more.